packages/wallet/src/useOnboard.tsx

```diff
@@ -44,7 +46,11 @@ const injectedWallets = new Set(['metamask', 'coinbase']);
 
 const injectedWallet = injectedModule();
 const ledger = ledgerModule();
-const walletConnect = walletConnectModule();
+const walletConnect = walletConnectModule({
+    projectId: WALLETCONNECT_PROJECT_ID || 'ba5de27656b7af8e2dab344f790ad81a',
```

Comment:
Do we really need this fallback here? Having it in the env vars is enough I think. Maybe we can leverage an [error boundary](https://react.dev/reference/react/Component#catching-rendering-errors-with-an-error-boundary) and present some error UI if the project id env var is not defined.

Reply:
@nevendyulgerov I provide the fallback because when I run the unit test it will return an error because the env vars is always return `undefined`. Do you think I should export the variable so in unit test we can add dummy `project_id`?